apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: VideoPlayer
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/VideoPlayer/VideoPlayer.tsx
+storybook: '/brand/storybook/?path=/story/components-videoplayer--playground'
+description: Use the video player component to present videos availible via CDN or local.
```

Comment:
ending at local feels a bit abrupt, how about this suggestion for conciseness...

```suggestion
description: Use the video player component to present self-hosted videos available via CDN or local.
```